sdk/identity/azure-identity/test/ut/client_certificate_credential_test.cpp

```diff
@@ -165,6 +165,269 @@ TEST(ClientCertificateCredential, Regular)
   EXPECT_LE(response1.AccessToken.ExpiresOn, response1.LatestExpiration + 7200s);
 }
 
+TEST(ClientCertificateCredential, AzureStack)
```

Comment:
What aspects of this test is related to Azure Stack? It isn't clear to me, so trying to understand better.

Reply:
In Azure Stack, TenantId == "adfs" instead of UUID, and URL path for the auth endpoint is slightly different.